packages/wallet/src/components/kyc/context.tsx

```diff
@@ -3,6 +3,8 @@ import { createContext, useContext, useState, type ReactNode } from 'react'
 type KYCFormContextProps = {
   tab: number
   setTab: (tab: number) => void
+  disabled: boolean
+  setDisabled: (isDisabled: boolean) => void
```

Comment:
The `disabled` state is a general state for the whole KYC or we need this to work per tab?

Reply:
Currently it is a general state, so that user doesn't navigate to ID Proof or Bank Account before feeling up Personal Details. In the future it might be a tab disabled. But the way it is currently working it is per whole group.

Reply:
Wouldn't this mean that the user is going to be able to go the first tab or the last when we change `disabled` to `false`? 
https://github.com/interledger/testnet/blob/5f050932fb23b5724316d56909507241795d4a4b/packages/wallet/src/components/kyc/PersonalDetailsForm.tsx#L44-L54